test/compiler.js

```diff
@@ -462,15 +476,15 @@ function runTests (solc, versionText) {
           }
         }
 
-        var output = JSON.parse(solc.compile(JSON.stringify(input), findImports));
+        var output = JSON.parse(solc.compile(JSON.stringify(input), wrapImportCallbackVersion(solc, findImports)));
         var x = getBytecodeStandard(output, 'cont.sol', 'x');
         st.ok(typeof x === 'string');
         st.ok(x.length > 0);
         var L = getBytecodeStandard(output, 'lib.sol', 'L');
         st.ok(typeof L === 'string');
         st.ok(L.length > 0);
 
-        var outputNewApi = JSON.parse(solc.compile(JSON.stringify(input), { import: findImports }));
+        var outputNewApi = JSON.parse(solc.compile(JSON.stringify(input), wrapImportCallbackVersion(solc, findImports)));
```

Comment:
These two (`solc.compile`) should not use the wrapper, since the goal of their interface is to hide this.

Reply:
Right

Reply:
Ah, but the interface hides this based on the version of the used `soljson`. If you use the new `soljon`, this breaks without this version wrapper